executor/aggfuncs/spill_deserialize_helper.go

```diff
@@ -0,0 +1,252 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+import (
+	"time"
+	"unsafe"
+
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/hack"
+	"github.com/pingcap/tidb/util/spill"
+)
+
+const intLen = int64(unsafe.Sizeof(int(0)))
+const boolLen = int64(unsafe.Sizeof(true))
+const uint32Len = int64(unsafe.Sizeof(uint32(0)))
+const uint64Len = int64(unsafe.Sizeof(uint64(0)))
+const int8Len = int64(unsafe.Sizeof(int8(0)))
+const int32Len = int64(unsafe.Sizeof(int32(0)))
+const int64Len = int64(unsafe.Sizeof(int64(0)))
+const float32Len = int64(unsafe.Sizeof(float32(0)))
+const float64Len = int64(unsafe.Sizeof(float64(0)))
+const timeLen = int64(unsafe.Sizeof(types.Time{}))
+const durationLen = int64(unsafe.Sizeof(time.Duration(0)))
+
+type strSizeType uint16
+
+type spillDeserializeHelper struct {
```

Comment:
We can move the spillSerixxxHelper and spillDeserixxxHelper to the same file, and rename the file as agg_spill_handler.go.

Reply:
> We can move the spillSerixxxHelper and spillDeserixxxHelper to the same file, and rename the file as agg_spill_handler.go.

I believe two file is more clear and reading codes is more easier.